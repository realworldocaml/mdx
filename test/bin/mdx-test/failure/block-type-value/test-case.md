This tests that erros are properly reported when the `type` label
is misused.

It requires a value

<!-- $MDX type -->
```ocaml
```

It only accepts a fixed set of values

<!-- $MDX type=invalid -->
```ocaml
```
