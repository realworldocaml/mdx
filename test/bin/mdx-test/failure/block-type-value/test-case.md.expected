[mdx] Fatal error: File "test-case.md", lines 6-8: invalid code block: Label `type` requires a value.
[mdx] Fatal error: File "test-case.md", lines 12-14: invalid code block: "invalid" is not a valid value for label `type`. Valid values are "ocaml", "cram", "toplevel" and "include".
